src/UglyToad.PdfPig/Actions/ActionProvider.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace UglyToad.PdfPig.Actions
+{
+    using Core;
+    using Logging;
+    using Outline;
+    using Tokenization.Scanner;
+    using Tokens;
+    using Outline.Destinations;
+    using Util;
+
+    internal static class ActionProvider
+    {
+        /// <summary>
+        /// Get an action (A) from dictionary. If GoTo, GoToR or GoToE, also fetches the action destination.
+        /// </summary>
+        /// <param name="dictionary"></param>
```

Comment:
Imo it's better to remove these empty `<param` items from the doccomments than leave them since it just adds to visual noise without any benefit